config/environments/development.rb

```diff
@@ -13,6 +13,8 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
+  config.hosts.clear
```

Comment:
Why did you need to add this? May be worth adding a comment for posterity

Reply:
This is a solution that I found to an error resulting from my computer not being on the allowed hosts list